masonry/src/core/contexts.rs

```diff
@@ -777,6 +777,11 @@ impl_context_method!(
         pub fn to_window(&self, widget_point: Point) -> Point {
             self.widget_state.window_transform * widget_point
         }
+
+        /// Get DPI scaling factor.
+        pub fn get_scale_factor(&self) -> f64 {
```

Comment:
Why is this made available for `ComposeCtx`? If the scale factor changes, as it currently stands, `compose` won't be re-run, so using this in `ComposeCtx` would lead to stale results.

Reply:
Excluded ComposeCtx

Reply:
Thanks! I think the set provided there is mostly reasonable.